homeassistant/components/zestimate/sensor.py

```diff
@@ -70,7 +70,7 @@ def __init__(self, name, params):
     @property
     def unique_id(self):
         """Return the ZPID."""
-        return self.params['zpid']
+        return '{} {}'.format(self._name, self.address)
```

Comment:
We shouldn't rely on name for a unique ID, names can change.

Reply:
oops, yeah, my bad.  thats what I get for being in a hurry.  Ill fix and re-push

thx